### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by NeKosmico** [WhiskeySockets/Baileys#758]
![IMG_20240430_214127](https://github.com/WhiskeySockets/Baileys/assets/165345955/a8ac70ff-bbf3-4de9-b611-4883a9698f4e)

**I get this error and I can't find a solution :'/**

- My code:
```javascript
const store = makeInMemoryStore({ logger: pino().child({ level: "silent", stream: "store" }) });

```

**Comment by tallw** [comment]
https://github.com/WhiskeySockets/Baileys/issues/747
Solution

**Comment by yogadev1990** [comment]
same issue

**Comment by PurpShell** [comment]
fixed in 6.7.2
